game-core/src/main/java/games/strategy/util/Version.java

```diff
@@ -76,8 +66,8 @@ public Version(final String version) {
    * Returns the exact and full version number.
    * For example, if we specify:
    * <code>
-   * new Version(1.2.3.4.5).getMicro == 4; // true
-   * new Version(1.2.3.4.5).toString().equals("1.2.3.4"); // true
+   * new Version(1.2.3).getPoint == 3; // true
```

Comment:
I think the purpose of this comment was to demonstrate that anything after the last component was ignored by the parser and would not affect the component accessors, so using the same version string as the other examples still seems appropriate:

```suggestion
   * new Version(1.2.3.4.5).getPoint == 3; // true
```

Since `micro` has been removed, it might make sense to change all the examples to use `1.2.3.4` as input instead of `1.2.3.4.5`.